build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
```

Comment:
You could also change the starter from webflux to web

Reply:
Yeah, I was thinking about this. The only reason I hadn't yet is that it then becomes an interesting exercise to refactor it _back_ to reactive if anyone is interested/prefers that.

But, I guess there's not going to be enough time in the workshop, so maybe not...

Reply:
IMO since this workshop is focused on testing, not writing the apps, making it an exercise will only distract from the topic

Reply:
Yeah, thinking about this more, I agree (_having the tests_ was amazingly useful for checking that I hadn't screwed anything up 😆).

Will update.